runtime/onert/backend/cpu/DynamicTensorManager.cc

```diff
@@ -75,6 +75,25 @@ void DynamicTensorManager::changeShape(const ir::OperandIndex &ind, const ir::Sh
   tensor->set_dynamic();
 }
 
+void DynamicTensorManager::planDealloc(const ir::Operation *op, ir::OperandIndex operand_ind)
+{
+  (void)op;
+  (void)operand_ind;
```

Comment:
(Optional) You can use another way to not write the name of parameters.

Reply:
You mean `UNUSED_RELEASE`?